sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryBuilderSamples.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
```

Comment:
Once we make the ADT query object public, this file will need to move to the samples project.